lib/send-html.js

```diff
@@ -0,0 +1,47 @@
+// Copyright IBM Corp. 2016. All Rights Reserved.
+// Node module: strong-error-handler
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+var ejs = require('ejs');
+var read = require('fs').readFileSync;
+var path = require('path');
+
+var assetDir = path.resolve(__dirname, '../views');
+var compileTemplate = {
```

Comment:
I think `compiledTemplates` is a better name. "compileTemplate" starts with a verb, names like that are used for functions/methods.